electron/storage.js

```diff
@@ -199,19 +217,31 @@ async function pushGameSaveToSteamCloud(base64save, currentPlayerId) {
   }
 }
 
+/**
+ * This function processes the save file in Steam Cloud and returns the save data in the base64 format. I don't change
+ * it to use the binary format, so there is no change in the logic of processing Steam Cloud save file.
+ */
 async function getSteamCloudSaveString() {
   if (!isCloudEnabled()) return Promise.reject("Steam Cloud is not Enabled");
   log.debug(`Fetching Save in Steam Cloud`);
   const cloudString = await getCloudFile();
-  const gzippedBase64Buffer = Buffer.from(cloudString, "base64");
-  const uncompressedBuffer = await gunzip(gzippedBase64Buffer);
+  // Decode cloudString to get compressedBuffer back.
+  const compressedBuffer = Buffer.from(cloudString, "base64");
+  // uncompressedBuffer is the binary form of the json save string.
+  const uncompressedBuffer = await gunzip(compressedBuffer);
```

Comment:
Wait, so cloud saves are *already* just the gzipped save string? XD

Reply:
Yes, the save file in Steam Cloud is the gzipped json save string.